wandb/sdk/artifacts/artifact.py

```diff
@@ -510,11 +498,13 @@ def metadata(self, metadata: dict) -> None:
 
     @property
     def ttl(self) -> Union[timedelta, None]:
```

Comment:
One thing that should be noted for the method that sets the TTL is that if a team admin has disallowed members from setting a TTL (see docs [here](https://docs.wandb.ai/guides/artifacts/ttl#define-who-can-edit-and-set-ttl-policies)), if they programmatically set a TTL it will not be set (results in a no-op) - i.e if you don't have permissions to set a TTL, you will not be able to successfully set it. cc @estellazx

Reply:
I added this reminder in the TTL setter